.github/rust.yml

```diff
@@ -0,0 +1,23 @@
+name: Rust
```

Comment:
Change workflow name to "Rust unit tests" mb?

Reply:
why `Rust ...` ?
I think better to name it `bAsset vault unit tests`